plugins/polio/api.py

```diff
@@ -2021,6 +2022,34 @@ def list(self, request):
         return Response({"results": results})
 
 
+@swagger_auto_schema(tags=["lqasimcountries"])
+class CountriesWithLqasIMConfigViewSet(ModelViewSet):
+    http_method_names = ["get"]
+    permission_classes = [permissions.IsAuthenticatedOrReadOnly]
+    results_key = "results"
+    serializer_class = OrgUnitDropdownSerializer
+    ordering_fields = ["name", "id"]
+    filter_backends = [
+        filters.OrderingFilter,
+        DjangoFilterBackend,
+    ]
+
+    def get_queryset(self):
+        category = self.request.query_params.get("category")
+        configs = Config.objects.filter(slug=f"{category}-config").first().content
+        country_ids = [
+            (config["country_id"] if JsonDataStore.objects.filter(slug=f"{category}_{config['country_id']}") else None)
+            for config in configs
+        ]
```

Comment:
@hakifran we can have an array containing none value then. @mathvdh any proposal to fix this ?

Reply:
@beygorghor I refactored the code and for now when the country has not datastore I do a continue in the loop.